test/test_utils/ec2.py

```diff
@@ -458,9 +459,10 @@ def execute_ec2_training_test(
     # Run training command
     shm_setting = '--shm-size="1g"' if large_shm else ""
     network = '--network="host" ' if host_network else ""
+    bin_bash_cmd = "--entrypoint /bin/bash " if bin_bash_entrypoint else ""
```

Comment:
Does the EC2 job run is needed as `execute_ec2_training_test` is used for EC2 test as well?

Reply:
Ran one revision with EC2 tests https://github.com/aws/deep-learning-containers/pull/1370/commits/0d7d89e501641e5c306a4d0c18cce3163ace8891 - will ensure this passes before merge